lib/bitset.js

```diff
@@ -2,102 +2,129 @@
 //
 //
 
-// A bitset implementation, after that in java.util.  Yes there
-// already exist such things, but none implement next{Clear|Set}Bit or
-// equivalent, and none involved me tooling about for an evening.
-
 'use strict';
 
-function BitSet(size) {
-  if (size) {
-    var numWords = Math.ceil(size / 32);
-    this.words = new Array(numWords);
+/**
+ * A bitset implementation, after that in java.util.  Yes there
+ * already exist such things, but none implement next{Clear|Set}Bit or
+ * equivalent, and none involved me tooling about for an evening.
+ */
+class BitSet {
```

Comment:
no need to adjust production code to call this with "new"?

Reply:
I looked at places where it did call `BitSet` in the source and all the places used `new BitSet`